src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -374,19 +393,10 @@ public void appendNodes(List<FlutterLogEntry> entries) {
         return;
       }
 
-      uiThreadAlarm.cancelAllRequests();
       uiThreadAlarm.addRequest(() -> {
```

Comment:
@pq, What do you think if we create a buffer to store the entries in this method (`appendNodes`)

And we will create a new thread to trigger that buffer each (10ms).